src/freenet/node/NodeCrypto.java

```diff
@@ -495,18 +443,11 @@ private String ecdsaSignRef(String mySignedReference) throws NodeInitException {
 
 	void addPrivateFields(SimpleFieldSet fs) {
 	    // Let's not add it twice
-	    fs.removeSubset("ecdsa");
-	    fs.put("ecdsa", ecdsaP256.asFieldSet(true));
```

Comment:
I've replaced the four spaces with two tabs... to be consistent with the rest of the file